lib/owncloud/test_moveFilesTwice.py

```diff
@@ -0,0 +1,106 @@
+
+__doc__ = """
+
+This test is testing that moving files server multiple do not get uploaded
+too much. Even if the files are moved when the sync is running.
+[https://github.com/owncloud/client/issues/4370]
+
+"""
+
+from smashbox.utilities import *
+import subprocess
+
+nfiles = 20
+TEST_FILES = ['test%02d.dat'%i for i in range(nfiles)]
+
+def getFileId(syncdir, fileName):
+    return subprocess.check_output(["sqlite3" , syncdir + "/.csync_journal.db",
+        "select fileid from metadata where path = \"" + fileName + "\""])
+
+@add_worker
+def workerA(step):
+    if compare_client_version('2.1.0', '<='):
```

Comment:
Well you can do:
```py
if not compare_client_version('2.1.1beta1', '>'):
```
so if it is not higher then beta1 you skip it.

Anyway clients should be updated fairly fast and we wont test beta1 forever I guess.